pkg/decision/evaluator/matchers/lt.go

```diff
@@ -18,27 +18,17 @@
 package matchers
 
 import (
-	"fmt"
-
-	"github.com/optimizely/go-sdk/pkg/decision/evaluator/matchers/utils"
 	"github.com/optimizely/go-sdk/pkg/entities"
+	"github.com/optimizely/go-sdk/pkg/logging"
 )
 
 // LtMatcher matches against the "lt" match type
 type LtMatcher struct {
 	Condition entities.Condition
+	Logger    logging.OptimizelyLogProducer
 }
 
 // Match returns true if the user's attribute is less than the condition's string value
 func (m LtMatcher) Match(user entities.UserContext) (bool, error) {
-
-	if floatValue, ok := utils.ToFloat(m.Condition.Value); ok {
-		attributeValue, err := user.GetFloatAttribute(m.Condition.Name)
-		if err != nil {
-			return false, err
-		}
-		return floatValue > attributeValue, nil
-	}
-
-	return false, fmt.Errorf("audience condition %s evaluated to NULL because the condition value type is not supported", m.Condition.Name)
+	return matchGtOrLt(user, m.Condition, m.Logger, false)
```

Comment:
Why we didn't only add log here.

Reply:
since both `gt` and `lt` matchers were very similar, linter was throwing `lines are duplicate` error.